tests/data_dummy/annotations/TitleC_with_custom_tag.xml

```diff
@@ -0,0 +1,13 @@
+<book title="TitleC">
```

Comment:
As characters appear in custom tags (`00000017` and `0000018`), such characters should be defined here beforehand, e.g.,
```
<characters>
  <character id="00000017" name="山田太郎"/>
  <character id="00000018" name="田中花子"/>
</characters>
```
See `TitleB.xml` for reference.

Reply:
I removed `character` tags intendedly, to show the custom case that does not have `characters.`
For this purpose, I should have removed the `character` attribute in 'body', 'custom_tag', and 'custom_tag2' also.

However, I guess it was not needed to show such a case here.
Thus, I added `character` tags, as you said. 
Thank you.

Reply:
I see. It may be better to cover all cases. Something like:
```xml
<!-- w/ character -->
<custom_tag id="00000019" attr_num="0" attr_str="aaa" attr_mix="123a" attr_unk='111' character="00000017"/>

<!-- w/o character -->
<custom_tag3 id="00000023" attr_num="234" attr_str="xxx" attr_mix="12fsd3a" attr_unk='13411'/>
```